bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -114,7 +114,9 @@ Equipment,Bed,,Bed,,
 Equipment,Blinds,,Blinds,"Rollershutter, Rollershutters, Roller shutter, Roller shutters, Shutter, Shutters",
 Equipment,Camera,,Camera,Cameras,
 Equipment,CleaningRobot,,Cleaning Robot,"Cleaning Robots, Vacuum robot, Vacuum robots",
+Equipment,CloudService,,Cloud Service,,
 Equipment,Computer,,Computer,,
+Equipment,Contact,,Contact,"Door Contact, Window Contact",
```

Comment:
I think that `Contact` should be a Point, not Equipment? The equipment would be `Door` and `Window`.

Reply:
> The equipment would be Door and Window

It is a magnetic contact device that could be fitted to a door or a window

Reply:
It is a separate magnet contact device that you can attach to a door or a window

Reply:
The magnetic contact device can be applied to so many types of equipment, but is meaningless on its own. 
`Contact` is not a device / equipment. It is a state / property.

Reply:
But if you are really so confused by the concept of `Equipment.CONTACT` as a piece of equipment we can call it  `Equipment.CONTACT_SENSOR` or `Equipment.CONTACT_DEVICE` instead, to make it clearer.

Reply:
`CONTACT_SESNOR` would be a good start. Could you point me to the comment / PR part about this rule you refer to. I need to catch up and have trouble finding the right posts, thanks!

Reply:
https://next.openhab.org/docs/developer/bindings/semantic-tags.html

Reply:
Thanks. `CONTACT_SENSOR` would be a good one to distinguish between the two.